From c458c8b4b2f9d7c73d69ea4c5b61493a11856b3d Mon Sep 17 00:00:00 2001
From: Nate Graham <nate@kde.org>
Date: Sun, 4 Aug 2019 09:43:11 -0600
Subject: [PATCH] Don't ship System Administration category anymore

Summary:
After D22929 lands, System Settings will ship the category itself. That's really the
proper place for it to live anyway.

Depends on D22929

Reviewers: #vdg, GB_2

Reviewed By: #vdg, GB_2

Differential Revision: https://phabricator.kde.org/D22930
---
 other/CMakeLists.txt                         |  1 -
 other/settings-system-administration.desktop | 43 --------------------
 2 files changed, 44 deletions(-)
 delete mode 100644 other/settings-system-administration.desktop

diff -Naur systemd-kcm-1.2.1/other/CMakeLists.txt systemd-kcm-1.2.1.tpg/other/CMakeLists.txt
--- a/other/CMakeLists.txt	2015-08-08 12:07:08.000000000 +0000
+++ b/other/CMakeLists.txt	2019-09-21 18:29:47.338793043 +0000
@@ -1,3 +1,2 @@
 install(FILES kcm_systemd.desktop DESTINATION ${CMAKE_INSTALL_KSERVICES5DIR})
-install(FILES settings-system-administration.desktop DESTINATION ${CMAKE_INSTALL_KSERVICES5DIR})
 
diff -Naur systemd-kcm-1.2.1/other/settings-system-administration.desktop systemd-kcm-1.2.1.tpg/other/settings-system-administration.desktop
--- a/other/settings-system-administration.desktop	2015-08-08 12:07:08.000000000 +0000
+++ b/other/settings-system-administration.desktop	1970-01-01 00:00:00.000000000 +0000
@@ -1,29 +0,0 @@
-[Desktop Entry]
-Type=Service
-X-KDE-ServiceTypes=SystemSettingsCategory
-X-KDE-System-Settings-Category=system-administration
-X-KDE-System-Settings-Parent-Category=
-X-KDE-Weight=200
-Icon=package-administration
-
-Name=System administration
-Name[bg]=Системна администрация
-Name[ca]=Administració del sistema
-Name[cs]=Administrace systému
-Name[de]=Systemverwaltung
-Name[es]=Administración del sistema
-Name[fi]=Järjestelmänhallinta
-Name[gl]=Administración do sistema
-Name[nl]=Systeemadministratie
-Name[pl]=Administracja systemu
-Name[pt]=Administração do sistema
-Name[pt_BR]=Administração do sistema
-Name[ro]=Administrare sistem
-Name[ru]=Системное администрирование
-Name[sk]=Správa systému
-Name[sl]=Skrbništvo sistema
-Name[sv]=Systemadministration
-Name[uk]=Адміністрування системи
-Name[x-test]=xxSystem administrationxx
-Name[zh_CN]=系统管理
-Name[zh_TW]=系統管理
-- 
2.21.0
